idb/postgres/postgres_integration_test.go

```diff
@@ -1103,3 +1103,210 @@ func TestOpenDbAgain(t *testing.T) {
 	_, err = OpenPostgres(connStr, idb.IndexerDbOptions{}, nil)
 	require.NoError(t, err)
 }
+
+func requireNilOrEqual(t *testing.T, expected string, actual *string) {
+	if expected == "" {
+		require.Nil(t, actual)
+	} else {
+		require.NotNil(t, actual)
+		require.Equal(t, expected, *actual)
+	}
+}
+
+// TestEmbeddedNullString make sure we're able to import cheeky assets.
+func TestNonDisplayableUTF8(t *testing.T) {
```

Comment:
Please add the B64 variation to this test ( or add another test that cover that functionality )

Reply:
This is testing the DB layer, so there is no B64 encoding to check here. In some cases there are the "B64" variables, which are actually byte arrays in the model, so I compare those with byte slice equality.

The tests in `test/common.sh` test the HTTP layer, but the test data is limited.